src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import "./utils/values.scss";
+@import "./card.scss";
+@import "./stars.scss";
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
```

Comment:
```suggestion

```
Consider moving general styles to a separate file to keep your main file clean and clear